website/src/data/components/line/props.ts

```diff
@@ -541,6 +541,15 @@ const props: ChartProperty[] = [
         defaultValue: defaults.enableTouchCrosshair,
         control: { type: 'switch' },
     },
+    {
+        key: 'initialHiddenIds',
+        flavors: allFlavors,
+        group: 'Interactivity',
+        help: `Hides certain series by default given their ids`,
+        type: 'boolean',
+        defaultValue: defaults.initialHiddenIds,
+        control: { type: 'switch' },
```

Comment:
No control should be defined, as the value cannot be changed via the demo.

Reply:
i've removed the control @plouc